packages/deploy-script-support/README.md

```diff
@@ -1,4 +1,94 @@
 # Deploy Script Support
 
-Import this code in your dapp deploy scripts to more easily install and
-interact with Zoe contracts.
\ No newline at end of file
+To install code on chain or in the a3p-integration environment, you'll have to
+generate a proposal, and write a script to build the core proposal. The
+proposals have limited access to bootstrap powers, described by their manifests.
+
+
+### Proposal
+
+The proposal is called with `(powers, options)` available. The manifest
+detailing the powers that will be used is usually in the same file, and
+conventionally provided by a function named `getManifestForFoo`. The manifest
+needs to have a unique name, since it will be referenced by name from the script.
+The usual format is
+```js
+export const foo = async (
+  {
+    consume: {
+      ...
+    },
+    brands: {
+      ...
+    }
+  },
+ options,
+) => {
+  // do the things using powers and options
+};
+
+export const getManifestForFoo = (powers, options) => {
+  manifest: {
+     [foo.name]: {
+       consume: {
+         ...
+      },
+  options,
+)};
+```
+
+`manifest` contains descriptions of powers to be provided to the proposals.
+
+**TODO**  what happens with the `installations` in [`startPsm.js`](https://github.com/Agoric/agoric-sdk/blob/b13743a2cccf0cb63a412b54384435596d4e81ea/packages/inter-protocol/src/proposals/startPSM.js#L496)?
+
+`options` allows the proposal to be provided with arbitray other powerful
+objects.
+
+### Script
+
+
+**REVIEWERS**: _I don't know how much of this applies to building proposals for
```

Comment:
a3p is running a synthetic mainnet, so it's the same either way.

remember to remove this comment before merge

Reply:
Thanks.